Makefile

```diff
@@ -172,19 +187,36 @@ $(shell mkdir -p build/baserom build/assets/text $(foreach dir,$(SRC_DIRS) $(UND
 build/src/libultra/libc/ll.o: OPTFLAGS := -Ofast
 build/src/%.o: CC := $(CC) -fexec-charset=euc-jp
 
+build/src/overlays/actors/ovl_Item_Shield/%.o: OPTFLAGS := -O2
```

Comment:
this can stay for now, though note that it's not a real fix, the actual bug comes from a DL issue with adult Zelda's object (if the crash you had was in the collapse sequence as child), I'll do a proper fix for that later

ah also we need to make sure those overrides doesn't affect N64 behavior

Reply:
N64 behavior isnt any different afaik. Was testing all yd but an actual fix would be cool.